app/quest-boost/[boostId]/page.tsx

```diff
@@ -129,55 +136,69 @@ export default function Page({ params }: BoostQuestPageProps) {
-              <p>Reward:</p>
-              <div className={questStyles.issuer}>
-                <p>
-                  {boost?.amount} {getTokenName(boost?.token ?? "")}
-                </p>
-                <TokenSymbol tokenAddress={boost?.token ?? ""} />
+          {boost && (
+            <>
+              <div className="flex flex-col">
+                <h1 className={styles.title}>{boost?.name}</h1>
+                {boost?.expiry && boost.expiry > Date.now() ? (
+                  <Timer fixed={false} expiry={Number(boost?.expiry)} />
+                ) : null}
+              </div>
+
+              <div className={styles.card_container}>
+                {quests?.map((quest, index) => {
+                  if (quest?.hidden || quest?.disabled) return null;
+                  return (
+                    <Quest
+                      key={index}
+                      title={quest.title_card}
+                      onClick={() => router.push(`/quest/${quest.id}`)}
+                      imgSrc={quest.img_card}
+                      issuer={{
+                        name: quest.issuer,
+                        logoFavicon: quest.logo,
+                      }}
+                      reward={quest.rewards_title}
+                      id={quest.id}
+                      expired={quest.expired}
+                    />
+                  );
+                })}
               </div>
-              <p>among</p>
-              <p className={styles.claim_button_text_highlight}>
-                {participants} players
-              </p>
-            </div>
-            {address ? (
-              <div>
-                <Button disabled={buttonDisabled} onClick={handleButtonClick}>
-                  {getButtonText()}
-                </Button>
+              <div className={styles.claim_button_container}>
+                <div className={styles.claim_button_text_content}>
+                  <p>Reward:</p>
+                  <div className={questStyles.issuer}>
+                    <p>
+                      {boost?.amount} {getTokenName(boost?.token ?? "")}
+                    </p>
+                    <TokenSymbol tokenAddress={boost?.token ?? ""} />
+                  </div>
+                  <p>among</p>
+                  <p className={styles.claim_button_text_highlight}>
+                    {participants} players
+                  </p>
+                </div>
+                {address ? (
+                  <div>
+                    <Button
+                      disabled={buttonDisabled}
+                      onClick={handleButtonClick}
+                    >
+                      {getButtonText()}
+                    </Button>
+                  </div>
+                ) : null}
               </div>
-            ) : null}
-          </div>
+            </>
+            )}
+            {
+              !boost && (<ErrorScreen
+                errorMessage="This quest doesn't exist !"
```

Comment:
in the issue you might see that i give the appropriate text which go here. Can you use those as string input here?